src/app/view/components/_common/dot-global-message/dot-global-message.component.scss

```diff
@@ -0,0 +1,26 @@
+@import "primeng-theme/utils/theme-variables";
+
+:host{
+    display: flex;
+
+    div:first-child{
+        opacity: 0;
+        transition:  $basic-speed * 2;
+    }
+
+    div.dot-global-message__visible{
+        opacity: 1;
+        transition:  $basic-speed * 2;
+    }
+
+    .dot-global-message__icon{
+        margin: 0 $basic-padding / 2;
+        font-size: 19px;
```

Comment:
There are variables for font sizes, check if there is one that fixes.

Reply:
there is none for 19px.